README.md

```diff
@@ -67,10 +67,16 @@ Navigate to the [chrome://extensions](chrome://extensions) page, enable
 
 ### Firefox
 
+For Firefox >48: Build the extensions with `WITH_ADDON_ID=true yarn build`
```

Comment:
Just to make sure: Is it >48 or ≥48 (a.k.a. 48+)?

Reply:
Also, how about naming the variable `WITH_GECKO_ID` which is less generic but closer to what actually happens under the hood?

Reply:
👍